providers/spotify/spotify.go

```diff
@@ -0,0 +1,27 @@
+package dumpert
+
+import (
+	"fmt"
+	"net/url"
+	"os/exec"
+
+	"github.com/zquestz/s/providers"
+)
+
+func init() {
+	providers.AddProvider("spotify", &SpotifyProvider{})
+}
+
+// SpotifyProvider adheres to the Provider interface.
+type SpotifyProvider struct {
+}
+
+// BuildURI generates a search URL for spotify.
+func (p *SpotifyProvider) BuildURI(q string) string {
+	_, err := exec.LookPath("spotify")
```

Comment:
On OSX this will be /Applications/Spotify.app, on windows I am not 100% sure, but probably spotify.exe. Need to check all those cases. =)